plugins/gcp-projects/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@backstage/plugin-gcp-projects",
+  "version": "0.1.1-alpha.16",
```

Comment:
Need to update these versions to the latest before we merge. This and the deps below.

Reply:
Updated dependencies :)